src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {

```

Comment:
The interface name `AreaCalculator` implies that it's responsible for calculating an area, which is correct. However, according to the checklist, we should avoid using verbs for class/interface names. A more appropriate name could be `AreaCalculable` to indicate that it provides an ability to calculate an area.